plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoClientConfig.java

```diff
@@ -44,6 +44,7 @@
     private Optional<String> connectionUrl = Optional.empty();
     private List<ServerAddress> seeds = ImmutableList.of();
     private List<MongoCredential> credentials = ImmutableList.of();
+    private boolean metadataPrivilegesRequired = true;
```

Comment:
Could you share why we want to introduce a config property? Other connectors hide unauthorized databases and tables. Always enabling `authorizedDatabasesOnly` & `authorizedCollections` properties looks fine to me.